### DISCUSSION TITLE
I can't find version.py anywhere

### DISCUSSION THREAD

**Discussion by tjing11** [blakeblackshear/frigate#4161] [Ask A Question] [answered]
I received this error message 
"ModuleNotFoundError: No module named 'frigate.version'". 
Should I add it myself or find it somewhere else in the repository? Thanks.

**Comment by NickM-27** [comment, marked as answer]
You just need to run `make version`

**Reply by tjing11** [reply]
Thanks!

**Reply by banthungprong** [reply]
In the Doc there is only "make". Had the same problem when cloning the rep for the first time.
Maybe adding "version" behind "local" in Makefile would solve this? local is the default and with this change
`local: version`
the "make" would generate the version.py.
